core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -312,6 +313,20 @@ List<ActiveCompaction> getActiveCompactions(ServerId server)
    */
   List<ActiveCompaction> getActiveCompactions() throws AccumuloException, AccumuloSecurityException;
 
+  /**
+   * List the active compaction running on a collection of TabletServers or Compactors. The server
+   * addresses can be retrieved using {@link #getCompactors()} or {@link #getTabletServers()}. Use
```

Comment:
The getCompactors and getTabletServers methods are deprecated, should point to the new getServers methods in the javadoc.